src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Staking withdrawals
+description: Page summarizing what staking push withdrawals are, how they work, and what stakers need to do to get their rewards
+lang: en
+template: staking
+image: ../../../assets/staking/leslie-withdrawal.png
+alt: Leslie the rhino with her staking rewards
+sidebarDepth: 2
+summaryPoints:
+  - The Shanghai upgrade brings staking withdrawals to Ethereum
+  - Validator operators must provide a withdrawal address to enable
+  - Rewards are automatically distributed every few days
+  - Validators who fully exit staking will receive their remaining balance
+---
+
+<UpgradeStatus dateKey="page-upgrades-withdrawals">
+  Staking withdrawals will be enabled through the Shanghai/Capella upgrade. This Ethereum network upgrade is expected to take place in the first half of 2023. <a href="#when">More below</a>
+</UpgradeStatus>
+
+The Shanghai/Capella upgrade enables **staking withdrawals** on Ethereum, allowing people to unlock ETH staking rewards. Reward payments will automatically and regularly be sent to a provided withdrawal address linked to each validator. Users can also exit staking entirely, unlocking their full validator balance.
+
+## Staking rewards {#staking-rewards}
+
+Reward payments are automatically processed for active validator accounts with a maxed out effective balance of 32 ETH, who are accumulating rewards.
```

Comment:
i see, hm  i am torn whats better because this sentence is trying to explain the general process so i thinks its better if it focuses on the majority use case and then elsewhere shed light on the side situations but i can see how we might not want to misguide people.